.github/workflows/linux.yml

```diff
@@ -204,13 +200,15 @@ jobs:
       OV_TOKENIZERS_INSTALL_DIR: ${{ github.workspace }}/openvino_tokenizers/install
       OPENVINO_TOKENIZERS_REPO: ${{ github.workspace }}/openvino_tokenizers
       BUILD_DIR: ${{ github.workspace }}/openvino_tokenizers/build
+      OV_PACKAGE: ${{ needs.openvino_download.outputs.ov_package_name || needs.openvino_build.outputs.ov_package_name }}
 
     steps:
       - name: Clone Openvino tokenizers
         uses: actions/checkout@v4
         with:
           path: ${{ env.OPENVINO_TOKENIZERS_REPO }}
 
+      # TODO: take install_build_dependencies from package?
```

Comment:
or even better - to reuse OpenVINO docker images

Reply:
I think we can replace it with install_openvino_dependencies (https://github.com/mryzhov/openvino.genai/blob/pybind_13/.github/workflows/linux.yml#L239-L240)

Reply:
but if we use self-hosted runners we can use our dockers too, agree, @akladiev  please use OV build docker image for tokenizers build. then  we can remove this step

Reply:
The whole pipeline needs to be rewritten to use Azure-based runners then, it's a separate task